core/domain/summary_services_test.py

```diff
@@ -134,3 +135,51 @@ def test_get_displayable_exp_summary_dicts_matching_ids(self):
         self.assertIn('last_updated_msec', displayable_summaries[0])
         self.assertDictContainsSubset(expected_summary,
                                       displayable_summaries[0])
+
+
+class FeaturedExplorationDisplayableSummaries(
+        test_utils.GenericTestBase):
+    """Test functions for getting displayable
+     featured exploration summary dicts."""
+
+    ALBERT_NAME = 'albert'
+    ALBERT_EMAIL = 'albert@example.com'
+
+    EXP_ID_1 = 'eid1'
+    EXP_ID_2 = 'eid2'
+
+    def setUp(self):
+        """Populate the database of explorations and their summaries.
+
+        The sequence of events is:
+        - (1) Albert creates EXP_ID_1.
+        - (2) Albert publishes EXP_ID_1.
+        - (3) Albert creates EXP_ID_2.
+        - (4) Albert publishes EXP_ID_2.
+        - (5) Admin user is set up.
+        """
+        super(FeaturedExplorationDisplayableSummaries, self).setUp()
+
+        self.admin_id = self.get_user_id_from_email(self.ADMIN_EMAIL)
+        self.albert_id = self.get_user_id_from_email(self.ALBERT_EMAIL)
+        self.signup(self.ADMIN_EMAIL, self.ADMIN_USERNAME)
+        self.signup(self.ALBERT_EMAIL, self.ALBERT_NAME)
+
+        self.save_new_valid_exploration(self.EXP_ID_1, self.albert_id)
+        self.save_new_valid_exploration(self.EXP_ID_2, self.albert_id)
+
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_1)
+        rights_manager.publish_exploration(self.albert_id, self.EXP_ID_2)
+
+        self.set_admins([self.ADMIN_USERNAME])
+
+    def test_for_featured_explorations(self):
+        # There are list of explorations
```

Comment:
Suggested docstring:

Note that EXP_ID_1 is public, and EXP_ID_2 is publicized. The call to get_featured_explorations() should only return [EXP_ID_2].